### PR TITLE
RyanairService basic implementation

### PR DESCRIPTION
**Pull request by KeyRotation**

### CODE REVIEW

File: app/src/main/java/com/braykoglab/xflights/model/Flight.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
+package com.braykoglab.xflights.model;
+
+
+import java.math.BigDecimal;
+import java.util.Date;
+
+public class Flight {
+    private BigDecimal flightPrice;
```
Comment (GlabPiotr) [inline app/src/main/java/com/braykoglab/xflights/model/Flight.java]: I would name it "price".

File: app/src/main/java/com/braykoglab/xflights/model/Flight.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    private Date flightDate;
```
Comment (GlabPiotr) [inline app/src/main/java/com/braykoglab/xflights/model/Flight.java]: I would name it "date"

File: app/src/main/java/com/braykoglab/xflights/model/Flight.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    private Currency priceCurrency;
```
Comment (GlabPiotr) [inline app/src/main/java/com/braykoglab/xflights/model/Flight.java]: Maybe just "currency" is better?

File: app/src/main/java/com/braykoglab/xflights/model/Flight.java [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+    public Flight(BigDecimal flightPrice, Date flightDate, Currency priceCurrency) {
+        this.flightPrice = flightPrice;
+        this.flightDate = flightDate;
+        this.priceCurrency = priceCurrency;
+    }
+
+    public BigDecimal getFlightPrice() {
```
Comment (GlabPiotr) [inline app/src/main/java/com/braykoglab/xflights/model/Flight.java]: Let`s use Lombok for getters and setter.s

File: app/src/main/java/com/braykoglab/xflights/MainActivity.java [hunk 5]
```diff
@@ -1,13 +1,58 @@
...
 import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+import android.widget.Toast;
 
-public class MainActivity extends AppCompatActivity {
+import com.braykoglab.xflights.service.RyanairApiService;
+
+public class MainActivity extends AppCompatActivity implements ServiceConnection {
+
+    private RyanairApiService ryanairApiService;
```
Comment (GlabPiotr) [inline app/src/main/java/com/braykoglab/xflights/MainActivity.java:17]: Isnt`t better to use simple name like: "RynairSerivce"?

File: app/src/main/java/com/braykoglab/xflights/model/ApiRequest.java [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+        this.to = to;
+        this.flyOutFrom = flyOutFrom;
+        this.flyOutTo = flyOutTo;
+        this.flyBackFrom = flyBackFrom;
+        this.flyBackTo = flyBackTo;
+        this.maxPrice = maxPrice;
+        this.currency = currency;
+    }
+
+    public static class ApiRequestBuilder {
```
Comment (GlabPiotr) [inline app/src/main/java/com/braykoglab/xflights/model/ApiRequest.java]: Please use Lombok @Builder annotation. It will be easier to maintain the code.

**Review by GlabPiotr — APPROVED** [review]

File: app/src/main/java/com/braykoglab/xflights/model/Flight.java (hunk 1 above)
Comment (KeyRotation) [reply app/src/main/java/com/braykoglab/xflights/model/Flight.java]: 👍

File: app/src/main/java/com/braykoglab/xflights/model/Flight.java (hunk 3 above)
Comment (KeyRotation) [reply app/src/main/java/com/braykoglab/xflights/model/Flight.java]: 👍

File: app/src/main/java/com/braykoglab/xflights/model/Flight.java (hunk 2 above)
Comment (KeyRotation) [reply app/src/main/java/com/braykoglab/xflights/model/Flight.java]: 👍

**Pull request merged**